lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -2280,14 +2280,17 @@ class ForumMFETestCase(ForumsEnableMixin, SharedModuleStoreTestCase):
     Tests that the MFE upgrade banner and MFE is shown in the correct situation with the correct UI
     """
 
+    features_with_enable_mfe_settings = settings.FEATURES.copy()
+    features_with_enable_mfe_settings['ENABLE_MFE_FOR_TESTING'] = True
```

Comment:
Maybe turning this variable a little bit more descriptive:

```suggestion
    FEATURES_WITH_DISCUSSION_MFE_ENABLED = settings.FEATURES.copy()
    FEATURES_WITH_DISCUSSION_MFE_ENABLED['ENABLE_MFE_FOR_TESTING'] = True
```